lib/Suggestions.js

```diff
@@ -86,6 +86,7 @@ class Suggestions extends Component {
             key={i}
             onMouseDown={props.handleClick.bind(null, i)}
             onMouseOver={props.handleHover.bind(null, i)}
+            onTouchStart={props.handleClick.bind(null, i)}
```

Comment:
this doesn't have good browser compatibility https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/ontouchstart
cc @mrdannael @prakhar1989 what do you suggest ?